apps/fz_http/lib/fz_http/gateways.ex

```diff
@@ -23,6 +47,24 @@ defmodule FzHttp.Gateways do
     |> Repo.delete()
   end
 
+  def get_gateway!, do: get_gateway!(name: @default_name)
   def get_gateway!(id: id), do: Repo.get!(Gateway, id)
   def get_gateway!(name: name), do: Repo.get_by!(Gateway, name: name)
+
+  def gateway_config(gateway) do
+    interface = %{
+      address: ["#{gateway.ipv4_address}/32", "#{gateway.ipv6_address}/128"],
```

Comment:
We used to do this, since the routing was handle differently(before the gateway) so I was keeping the same logic but I've changed that in the latest branch I'm working on to use the range from the network, since I need that to correctly route things in the gateway now.(please review that when I open the next PR)